infra/base-images/base-builder/Dockerfile

```diff
@@ -104,7 +104,7 @@ RUN mkdir honggfuzz && \
 COPY compile compile_afl compile_dataflow compile_libfuzzer compile_honggfuzz \
     precompile_honggfuzz srcmap write_labels.py /usr/local/bin/
 
-COPY detect_repo.py $SRC/
+COPY detect_repo.py /src
```

Comment:
why not `$SRC`? I think it should be present as it's already used on line 94.

Reply:
detect_repo.py is run to determine what the $SRC environment variable is, so detect_repo.py needs to be set at a known location so it can be run which is why the location is hardcoded.

@jonathanmetzman  and I discussed this as the alternative to getting SRC earlier from the docker container.

Reply:
I think this is fine for now, but let's put it in /opt in a later CL to avoid messing with user data in SRC and so that we don't need to worry about this problem (we can put the subdirectory in /opt in the PATH).